[2025-02-23T15:52:44.423+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T15:52:44.445+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.second_task scheduled__2025-02-22T00:00:00+00:00 [queued]>
[2025-02-23T15:52:44.456+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.second_task scheduled__2025-02-22T00:00:00+00:00 [queued]>
[2025-02-23T15:52:44.457+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-23T15:52:44.472+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): second_task> on 2025-02-22 00:00:00+00:00
[2025-02-23T15:52:44.480+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1566) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-23T15:52:44.482+0000] {standard_task_runner.py:72} INFO - Started process 1574 to run task
[2025-02-23T15:52:44.483+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'second_task', 'scheduled__2025-02-22T00:00:00+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmprvg_1xzu']
[2025-02-23T15:52:44.486+0000] {standard_task_runner.py:105} INFO - Job 57: Subtask second_task
[2025-02-23T15:52:44.535+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.second_task scheduled__2025-02-22T00:00:00+00:00 [running]> on host 2cebbf768e94
[2025-02-23T15:52:44.636+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-22T00:00:00+00:00'
[2025-02-23T15:52:44.637+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T15:52:44.655+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-23T15:52:44.656+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, this is task 2, and will be run after task 1.']
[2025-02-23T15:52:44.665+0000] {subprocess.py:99} INFO - Output:
[2025-02-23T15:52:44.666+0000] {subprocess.py:106} INFO - hey, this is task 2, and will be run after task 1.
[2025-02-23T15:52:44.667+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-23T15:52:44.693+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T15:52:44.694+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=second_task, run_id=scheduled__2025-02-22T00:00:00+00:00, execution_date=20250222T000000, start_date=20250223T155244, end_date=20250223T155244
[2025-02-23T15:52:44.741+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T15:52:44.774+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-23T15:52:44.777+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-24T04:47:22.991+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-24T04:47:23.018+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.second_task scheduled__2025-02-22T00:00:00+00:00 [queued]>
[2025-02-24T04:47:23.033+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.second_task scheduled__2025-02-22T00:00:00+00:00 [queued]>
[2025-02-24T04:47:23.033+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-24T04:47:23.052+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): second_task> on 2025-02-22 00:00:00+00:00
[2025-02-24T04:47:23.060+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1139) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-24T04:47:23.061+0000] {standard_task_runner.py:72} INFO - Started process 1157 to run task
[2025-02-24T04:47:23.060+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'second_task', 'scheduled__2025-02-22T00:00:00+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmp1_q2icra']
[2025-02-24T04:47:23.062+0000] {standard_task_runner.py:105} INFO - Job 102: Subtask second_task
[2025-02-24T04:47:23.120+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.second_task scheduled__2025-02-22T00:00:00+00:00 [running]> on host 79e0cb3ed452
[2025-02-24T04:47:23.216+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-22T00:00:00+00:00'
[2025-02-24T04:47:23.217+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-24T04:47:23.236+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-24T04:47:23.253+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, this is task 2, and will be run after task 1.']
[2025-02-24T04:47:23.271+0000] {subprocess.py:99} INFO - Output:
[2025-02-24T04:47:23.272+0000] {subprocess.py:106} INFO - hey, this is task 2, and will be run after task 1.
[2025-02-24T04:47:23.274+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-24T04:47:23.299+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-24T04:47:23.334+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=second_task, run_id=scheduled__2025-02-22T00:00:00+00:00, execution_date=20250222T000000, start_date=20250224T044723, end_date=20250224T044723
[2025-02-24T04:47:23.402+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-24T04:47:23.493+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-24T04:47:23.511+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
